test/manipulation.test.js

```diff
@@ -28,8 +33,11 @@ describe('manipulation', function() {
       age: {type: Number, index: true},
       dob: Date,
       createdAt: {type: Date, default: Date},
+      throwingSetter: {type: String},
```

Comment:
Should we set `default: null` too, for consistency?

Reply:
yes you are right,... forgot that ! Changing it now!